src/main/java/edu/kis/powp/jobs2d/visitor/ExceededCanvasVisitor.java

```diff
@@ -11,7 +11,7 @@
 public class ExceededCanvasVisitor implements ICommandVisitor{
 
     private Shape area;
-    public boolean exceeds = false;
+    public boolean exceeds = true;
```

Comment:
exceeds should be set to false directly before iteration in the 33 line while loop.